java/src/jmri/jmrix/bachrus/SpeedoConsoleFrame.java

```diff
@@ -823,9 +821,6 @@ public void initComponents() {
         //</editor-fold> 
 
         //<editor-fold defaultstate="collapsed" desc="Basic Speed Matching Tab">
-        basicSimpleCVSpeedMatchButton.setToolTipText(Bundle.getMessage("BasicSpeedMatchSimpleCVTooltip"));
-        basicSpeedTableSpeedMatchButton.setToolTipText(Bundle.getMessage("BasicSpeedMatchSpeedTableTooltip"));
-        basicESUSpeedMatchButton.setToolTipText(Bundle.getMessage("BasicSpeedMatchESUSpeedTableTooltip"));
```

Comment:
Why are these tooltips removed?

Reply:
I removed them because their content is discussed in the description above, which is more discoverable and easier to read. That and I didn't want to add corresponding tooltips for the speed step scale speed matcher.